docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/linux-packages.rst

```diff
@@ -77,18 +77,30 @@ Install PostgreSQL using ``apt`` or ``yum``
       sudo systemctl enable postgresql.service
 
 #. Configure a system account that Determined will use to connect to PostgreSQL. For example, to use
-   the default ``postgres`` user but update its password:
+   a user named ``determined`` with the password ``determined-password``:
 
    .. code::
 
       sudo -u postgres psql postgres
-      postgres=# \password postgres
+      postgres=# create user determined with encrypted password 'determined-password';
```

Comment:
Would it make sense to move this down to after hba.conf is configures, so the order is "configure auth, create user, create DB, give user access to DB"?  That way, all of the psql commands could be together in one block, or at least closer to one another.  I don't have strong feelings either way.

If you do move it, `sudo -u postgres psql postgres` doesn't really need a database name; it should work as just `sudo -u postgres psql.  But that's totally not hurting anything the way it is, either.`